fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/MetaFunctions.java

```diff
@@ -234,4 +239,63 @@ public static ConstantOperator inspectTaskRuns() {
         return ConstantOperator.createVarchar(trm.inspect());
     }
 
+    @ConstantFunction(name = "inspect_memory", argTypes = {VARCHAR}, returnType = VARCHAR, isMetaFunction = true)
```

Comment:
@mofeiatwork why put this class under `rewrite` package?

Reply:
I asked mofei and he agreed to change the directory.